web/vtadmin/src/components/routes/Keyspaces.tsx

```diff
@@ -91,7 +95,7 @@ export const Keyspaces = () => {
                     value={filter || ''}
                 />
                 <div className="max-w-screen-md">
-                    <DataTable columns={['Keyspace', 'Shards']} data={ksRows} renderRows={renderRows} />
+                    <DataTable columns={['Keyspace', 'Shards', 'Actions']} data={ksRows} renderRows={renderRows} />
```

Comment:
If it's not too much trouble, would you mind rebasing on https://github.com/vitessio/vitess/pull/9731 (when it's merged) and then hiding this column based on `isReadOnly()`? 🙇

Reply:
Yes 😄 Waiting for it to be merged!